distribution/changelog.txt

```diff
@@ -1,5 +1,7 @@
 0.4.16 (in development)
 ------------------------------------------------------------------------
+- Fix: [#22619] Splash Boats steep supports are drawn incorrectly.
+- Fix: [#22619] Side-Friction Roller Coaster steep supports are drawn incorrectly.
```

Comment:
Maybe these two entries could be merged into one?
`- Fix: [#22619] Side-Friction Roller Coaster and Splash Boats steep supports are drawn incorrectly.`